addons/web/static/src/js/chrome.js

```diff
@@ -252,6 +252,21 @@ instance.web.CrashManager = instance.web.Class.extend({
         if (!this.active) {
             return;
         }
+        if (error.code == -32098) {
+            $.blockUI({ message: '' , overlayCSS: {'z-index': 9999, backgroundColor: '#FFFFFF', opacity: 0.0, cursor: 'wait'}});
```

Comment:
@rim-odoo 
I Add the opacity because  dont want the black color overlay like second image.
If we want the overlay like current on runbot then we have to add this , otherwise it take opacity=0.6 and look like first image .
One more option
   If we want to black color overlay (like [2] image ) then we have to remove both 
            -backgroundColor=#FFFFFF
            -opacity=0

So what is your opinion ???

Without (opacity)
![screenshot1](https://cloud.githubusercontent.com/assets/7599690/6281784/863f252e-b8ed-11e4-8cf5-413d85b8e6ae.png)

Without (backgroundcolor and opacity)
![screenshot2](https://cloud.githubusercontent.com/assets/7599690/6281785/9126f4f8-b8ed-11e4-9c3a-73bca0c7c289.png)